charts/trustify/templates/services/importer/030-Deployment.yaml

```
@@ -67,6 +67,9 @@ spec:
                 labels:
                   type: importer-working-directory
               spec:
+                {{- with .Values.importer.storageClassName }}
+                storageClassName: {{ . | quote }}
+                {{- end }}
```

Since the importer is aligned under `module` section, we have to use `.Values.module.importer.storageClassName`. Since the $mod variable defined already, we can just refer like below,
```suggestion
                {{- with $mod.module.storageClassName }}
                storageClassName: {{ . | quote }}
                {{- end }}
```